glide.yaml

```diff
@@ -1,15 +1,19 @@
 package: k8s.io/federation
 import:
 - package: cloud.google.com/go
+  version: 3b1ae45394a234c385be014e9a488f2bb6eef821
```

Comment:
Any specific reason that we need to add the specific hash for this and many other repos? Probably @marun can comment with his experience of using glide earlier.

Reply:
just added the version to be extra careful to use the same version of the package used by corresponding kubernetes version. if we don't use version and not pinned to the same version used by kubernetes, there are possibilities that we might use the latest version of the package which might have syntactical or semantic changes which may cause problems for our build.

Reply:
I'm assuming glide should be able to specify the versions automatically?  I hope that it isn't necessary to hand-curate the versions of vendored dependencies, that seems onerous and error-prone.

Reply:
i think, glide does not fill the version automatically and always picks the latest commits in each package. It would be foolproof, if we pin the dependent package version to that of what is used in kubernetes version. There will be no compatibility issues and also kubernetes undergoes lot more testing than federation code. I agree that it is error-prone to hand-curate the versions, but its a one time activity and some 10-20 packages that we need to fill it by looking in Godeps.json file from corresponding kubernetes version.